tests/namespace/test_interfaces.py

```diff
@@ -626,11 +662,11 @@ def test_out_ucast_vlan(self):
         self.assertEqual(str(value0.name), str(ObjectIdentifier(11, 0, 1, 0, (1, 3, 6, 1, 2, 1, 2, 2, 1, 17, 3000))))
         self.assertEqual(value0.data, 20)
 
-    def test_out_errors_vlan(self):
```

Comment:
Minor comment, what is the expected result if ObjectIdentifier(11, 0, 0, 0, (1, 3, 6, 1, 2, 1, 2, 2, 1, 20, 3000)) is queried?
Can we keep the out_error_vlan check and check if the data is 0?

Reply:
@SuvarnaMeenakshi Hi, neither ifInDiscard nor ifOutDiscard was available for Vlan before this change. Since we moved errors to discards, now neither ifInErrors nor ifOutErrors OID exist for Vlan.
Do you prefer to just always get 0?

Reply:
ok got it, thank you for the details.

We can stick with option 1: "ifInErrors/ifOutErrors are unavailable for VLAN interface. This makes sense because there are no L1 errors on VLAN. This is the current behaviour with this PR."